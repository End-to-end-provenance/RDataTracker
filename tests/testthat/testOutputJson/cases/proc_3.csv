"ddg.type","ddg.num","ddg.name","ddg.value","ddg.return.linked","ddg.time","ddg.snum","ddg.startLine","ddg.startCol","ddg.endLine","ddg.endCol"
"Start",1,"ReturnTest.R","ReturnTest.R",FALSE,1.42,NA,NA,NA,NA,NA
"Operation",2,"f <- function(aa, bb) {    retValue <- f2(aa) + f2(bb)    re","f <- function(aa, bb) {    retValue <- f2(aa) + f2(bb)    re",FALSE,1.42,0,1,1,4,1
"Operation",3,"f2 <- function(p_a) {    return(10)}","f2 <- function(p_a) {    return(10)}",FALSE,1.43,0,6,1,8,1
