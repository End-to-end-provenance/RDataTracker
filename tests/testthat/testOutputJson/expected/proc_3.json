
	"activity" : {
		// procedure nodes
		"rdt:p1": {
			"rdt:name": "ReturnTest.R",
			"rdt:type": "Start",
			"rdt:elapsedTime": 1.42,
			"rdt:scriptNum": "NA",
			"rdt:startLine": "NA",
			"rdt:startCol": "NA",
			"rdt:endLine": "NA",
			"rdt:endCol": "NA"
		},
		"rdt:p2": {
			"rdt:name": "f <- function(aa, bb) {\tretValue <- f2(aa) + f2(bb)\tre",
			"rdt:type": "Operation",
			"rdt:elapsedTime": 1.42,
			"rdt:scriptNum": 0,
			"rdt:startLine": 1,
			"rdt:startCol": 1,
			"rdt:endLine": 4,
			"rdt:endCol": 1
		},
		"rdt:p3": {
			"rdt:name": "f2 <- function(p_a) {\treturn(10)}",
			"rdt:type": "Operation",
			"rdt:elapsedTime": 1.43,
			"rdt:scriptNum": 0,
			"rdt:startLine": 6,
			"rdt:startCol": 1,
			"rdt:endLine": 8,
			"rdt:endCol": 1
		}
	}
